services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsWaypoint.java

```diff
@@ -33,31 +34,32 @@ public static Builder builder() {
    * @return string with the name of the way the coordinate snapped to
    * @since 1.0.0
    */
-  @Nullable
+  @NonNull
```

Comment:
This is technically SemVer although we could treat it as a bug.

Reply:
That's my thinking, not a semver for the getter only for the setter, but more of a bug nonetheless.